scripts/check_configs.ts

```diff
@@ -31,6 +31,7 @@ interface ConfigJSON {
   wswth_contract?: string;
   market_banners?: MarketBanner[];
   market_promo?: {[marketId: string]: MarketPromo};
+  spot_pool_config: SpotPoolConfig;
```

Comment:
According to `config.schema.json`, `spot_pool_config` is optional, as such this field needs to be optional too.